packages/edit-site/src/store/private-actions.js

```diff
@@ -80,20 +86,22 @@ export const removeTemplates =
 		if ( promiseResult.every( ( { status } ) => status === 'fulfilled' ) ) {
 			let successMessage;
 
-			if ( templates.length === 1 ) {
+			if ( items.length === 1 ) {
 				// Depending on how the entity was retrieved its title might be
 				// an object or simple string.
-				const templateTitle =
-					typeof templates[ 0 ].title === 'string'
-						? templates[ 0 ].title
-						: templates[ 0 ].title?.rendered;
+				const title =
+					typeof items[ 0 ].title === 'string'
+						? items[ 0 ].title
+						: items[ 0 ].title?.rendered;
 				successMessage = sprintf(
 					/* translators: The template/part's name. */
 					__( '"%s" deleted.' ),
-					decodeEntities( templateTitle )
+					decodeEntities( title )
 				);
 			} else {
-				successMessage = __( 'Templates deleted.' );
+				successMessage = isTemplate
+					? __( 'Templates deleted.' )
```

Comment:
I've mentioned this elsewhere, but I think we could do with using a generic language. For example, `Items deleted.`.

The users already have the context of what they are – they know whether they are working with templates or template parts, don't need reminding. And this code would be a lot simpler.

I don't have a strong opposition to doing it this way, and it works fine. I just want to share an alternative that I find simpler.

Reply:
I agree  with this and would need less code :). Le't handle this in a follow up though because others might chip in about the clarity of the messages and wouldn't want to block  this PR for that.